src/neo-vm/ExecutionEngine.cs

```diff
@@ -361,7 +368,15 @@ private void ExecuteInstruction()
                         ExecutionContext context_pop = InvocationStack.Pop();
                         EvaluationStack stack_eval = InvocationStack.Count == 0 ? ResultStack : InvocationStack.Peek().EvaluationStack;
                         if (context_pop.EvaluationStack != stack_eval)
-                            context_pop.EvaluationStack.CopyTo(stack_eval);
+                        {
+                            int rvcount = context_pop.RVCount;
+                            if (rvcount < 0)
+                                rvcount = context_pop.EvaluationStack.Count;
+                            else if (context_pop.EvaluationStack.Count != rvcount)
```

Comment:
These checks will also be done in `EvaluationStack.CopyTo`

Reply:
Done.